### DISCUSSION TITLE
Jupyter notebook error

### DISCUSSION THREAD

**Discussion by aayushe** [microsoft/vscode-jupyter#13472] [Questions and Answers]
error :Error loading webview: Error: Could not register service worker: InvalidStateError: Failed to register a ServiceWorker: The document is in an invalid state.

**Comment by DonJayamanne** [comment]
@mjbvz /cc

**Comment by mjbvz** [comment]
https://github.com/microsoft/vscode/issues/125993 (just search for `InvalidStateError` in vscode issues)
